spec/controllers/webauth_controller_spec.rb

```diff
@@ -30,6 +30,22 @@
     end
   end
 
+  describe '#login_media_download' do
+    subject { get :login_media_download, id: 'bb582xs1304', file_name: 'bb582xs1304_sl', format: 'mp4' }
+
+    it 'returns the user to the media download action' do
+      expect(subject).to redirect_to media_url(controller.params.symbolize_keys)
+    end
+  end
+
+  describe '#login_media_stream' do
+    subject { get :login_media_stream, id: 'bb582xs1304', file_name: 'bb582xs1304_sl.mp4', format: 'm3u8' }
```

Comment:
in this case, it is a "format" for the concept of the manifest for the container.  it is implying   playlist.m3u8   for hls, or  manifest.mpd for dash.